Orange/widgets/classify/owadaboost.py

```diff
@@ -0,0 +1,76 @@
+from PyQt4.QtCore import Qt
+
+from Orange.classification.base_classification import LearnerClassification
+from Orange.data import Table
+from Orange.classification import TreeLearner
+from Orange.ensembles import SklAdaBoostLearner
+from Orange.widgets import gui
+from Orange.widgets.settings import Setting
+from Orange.widgets.utils.owlearnerwidget import OWBaseLearner
+
+
+class OWAdaBoostClassification(OWBaseLearner):
+    name = "AdaBoost"
+    description = "An AdaBoost classifier."
+    icon = "icons/AdaBoost.svg"
+    priority = 80
+
+    LEARNER = SklAdaBoostLearner
+    OUTPUT_MODEL_NAME = "Classifier"
+
+    inputs = [("Classifier", LearnerClassification, "set_base_model")]
```

Comment:
Classifier is name of the outputs that contain trained *model*, but here, a learner is expected. Should this be renamed to learner?